### PR TITLE
Document upcoming COVID vaccine acceptance signal

### PR DESCRIPTION
**Pull request by capnrefsmmat**
Currently preparing this signal for public release, so I'd like the docs to be ready when that happens (perhaps Wed Jan 6).

### CODE REVIEW

File: docs/api/covidcast-signals/fb-survey.md [hunk 1]
```diff
@@ -315,6 +317,23 @@ also available. These have names beginning `smoothed_w`, such as
...
+| `smoothed_accept_covid_vaccine` | Estimated percentage of respondents who would definitely or probably choose to get vaccinated, if a COVID-19 vaccine were offered to them today | V3 |
+
+This indicator is based on questions added in Wave 6 of the survey, introduced
+on December 19, 2020. **Note:** As of January 2021, vaccination items on the
+survey are being revised in preparation for Wave 7. The signal above may be
+replaced with a new signal (with different name) if the underlying survey item
+changes significantly.
+
+A weighted versions of this signal, using the [survey weighting described
+below](#survey-weighting) to be more representative of state demographics, is
```
Comment (Akvannortwick) [inline docs/api/covidcast-signals/fb-survey.md:333]:
```suggestion
below](#survey-weighting) to represent more state demographics, is
```
It's not major, just felt overly wordy the other way.

File: docs/api/covidcast-signals/fb-survey.md [hunk 2]
```diff
@@ -315,6 +317,23 @@ also available. These have names beginning `smoothed_w`, such as
...
+
+| Signal | Description | Survey Item |
+| --- | --- | --- |
+| `smoothed_accept_covid_vaccine` | Estimated percentage of respondents who would definitely or probably choose to get vaccinated, if a COVID-19 vaccine were offered to them today | V3 |
+
+This indicator is based on questions added in Wave 6 of the survey, introduced
+on December 19, 2020. **Note:** As of January 2021, vaccination items on the
+survey are being revised in preparation for Wave 7. The signal above may be
+replaced with a new signal (with different name) if the underlying survey item
+changes significantly.
```
Comment (Akvannortwick) [inline docs/api/covidcast-signals/fb-survey.md:330]:
```suggestion
survey are being revised in preparation for Wave 7. We may replace the signal above with a new signal (with a different name) if the underlying survey item changes significantly.
```

I would use an active voice, but if it's not terribly important no worries. The 'a' before different should be kept though.

File: docs/api/covidcast-signals/fb-survey.md (hunk 1 above)
Comment (krivard) [reply docs/api/covidcast-signals/fb-survey.md:333]: "representative of" is a statistical term of art; "represent" doesn't mean the same thing (sorry, English is garbage sometimes)

File: docs/api/covidcast-signals/fb-survey.md [hunk 3]
```diff
@@ -315,6 +317,23 @@ also available. These have names beginning `smoothed_w`, such as
...
+
+A weighted versions of this signal, using the [survey weighting described
```
Comment (krivard) [inline docs/api/covidcast-signals/fb-survey.md:332]:
```suggestion
A weighted version of this signal, using the [survey weighting described
```

**Review by krivard — APPROVED** [review]
typo, otherwise lgtm

File: docs/api/covidcast-signals/fb-survey.md (hunk 1 above)
Comment (Akvannortwick) [reply docs/api/covidcast-signals/fb-survey.md:333]: That's fair, no worries.

**Comment by capnrefsmmat** [comment]
Thanks. I merged the suggestions, so once the data is ready I'll merge this PR.

**Comment by capnrefsmmat** [comment]
Updated to indicate that as of today, the question is only shown to respondents who say they haven't been vaccinated. I'll merge this as soon as I see the data in the API and it looks right.

**Pull request merged**